.ci/run_nccl_test.sh

```diff
@@ -160,8 +161,9 @@ fi
     echo_hash_line
 
     MPIRUN_OPTIONS_SPECIFIC="\
-    -x NCCL_LL_THRESHOLD=1000000000 \
-    -x NCCL_TREE_THRESHOLD=1000000000 \
+    -x NCCL_PROTO=LL \
+    -x NCCL_ALGO=CollNet \
+    -x NCCL_COLLNET_ENABLE=1 \
```

Comment:
As far as I see it is not yet documented - what's the difference between  `NCCL_ALGO=CollNet + NCCL_COLLNET_ENABLE=1` and `NCCL_ALGO=CollNet`?

Reply:
Just NCCL_COLLNET_ENABLE=1 enables collective plugins support in NCCL, but NCCL can choose any available algorithm. By setting both flags we enable collective network support (NCCL_COLLNET_ENABLE=1 )and also force nccl to use CollNet for any message size (NCCL_ALGO=CollNet).